packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -0,0 +1,85 @@
+import Component from '@glimmer/component';
+import { service } from '@ember/service';
+import MonacoService from '@cardstack/host/services/monaco-service';
+import { trackedFunction } from 'ember-resources/util/function';
+import CardService from '@cardstack/host/services/card-service';
+import { htmlSafe } from '@ember/template';
+import OperatorModeStateService from '@cardstack/host/services/operator-mode-state-service';
+import type { Card } from 'https://cardstack.com/base/card-api';
+
+interface Signature {
+  Args: {
+    card: Card;
+  };
+}
+
+export default class OperatorModeCodeMode extends Component<Signature> {
```

Comment:
Can we make the class name simpler? Such as `CodeMode`.

Reply:
Yeah `OperatorModeCodeMode` really does sound awkward. Will shorten it